agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml

```diff
@@ -54,19 +54,24 @@ spec:
       - name: varrun-vol
         hostPath:
           path: /var/run
-      # Uncomment these lines if you'd like to map /root/ from the
-      # host into the container. This can be useful to map
-      # /root/.sysdig to pick up custom kernel modules.
-      #- name: host-root-vol
-      #  hostPath:
-      #    path: /root
+### Uncomment these lines if you'd like to map /root/ from the
+#   host into the container. This can be useful to map
+#   /root/.sysdig to pick up custom kernel modules.
+#     - name: host-root-vol
+#       hostPath:
+#         path: /root
```

Comment:
I'm not sure what the point of this change is...I'm not opposed to it, but if we want to go this route with the formatting we should be consistent throughout the file (lines 70-74, for instance, should follow the same pattern of column 1 comment marker and three comment markers at the start of the block comment). Either is fine, just make it consistent.

Reply:
Add jank to work around our other jank. Got it ;)

Fortunately, this jank is extremely low-magnitude. Thanks for the explanation!